### PR TITLE
Do size

### PR DESCRIPTION
**Pull request by tjj2017**
Handle S'Size and  X'Size Attributes.

### CODE REVIEW

File: gnat2goto/driver/tree_walk.adb [hunk 1]
```diff
@@ -5840,7 +5904,7 @@ package body Tree_Walk is
                                          "Unsupported pragma: Initializes");
          when Name_Annotate |
             --  Ignore here. Rather look for those when we process a node.
-              Name_Assertion_Policy |
+              Name_Assertion_Policy | Name_Check_Policy |
```
Comment (martin-cs) [inline gnat2goto/driver/tree_walk.adb]: Is this related?

**Comment by tjj2017** [comment]
The Assertion_Policy pragma was covered by gnat2goto but in fact it never occurs in the tree because the front end rewrites it to a Check_Policy pragma.  It was just found in passing.  Not really related to the handling of the size attribute.
